### ISSUE TITLE
Weight decay term reduces the accuracy significantly

### ISSUE THREAD

**Issue by thushv89** [thushv89/AutoEncorder_Simple#18]

**Comment by thushv89** [comment]
One possible explanation
http://stats.stackexchange.com/questions/117622/weight-decay-in-neural-neural-networks-weight-update-and-convergence

**Comment by thushv89** [comment]
Tests 
Without weight decay
Hidden [625,400,225]   Pre epochs 25 Fine-tune 50 batch_size 10
Test Error: 0.0168

With weight decay
Hidden [625,400,225]   Pre epochs 25 Fine-tune 50 batch_size 10
Test Error: 0.0169
